src/index.ts

```diff
@@ -70,6 +70,16 @@ export { default as Link, LinkTypes } from "./components/Link/Link";
 export { default as List, ListTypes } from "./components/List/List";
 export { default as Logo, LogoNames, LogoSizes } from "./components/Logo/Logo";
 export { ModalTrigger, useModal } from "./components/Modal/Modal";
+
+export {
+  default as MultiSelect,
+  MultiSelectProps,
+} from "./components/MultiSelect/MultiSelect";
+export {
+  MultiSelectItem,
+  SelectedItems,
+} from "./components/MultiSelect/MultiSelectTypes";
+
```

Comment:
@isastettler We also want to export the MultiSelect hook for consuming apps to use, so we need to add this as well here:

```
export { default as useMultiSelectState } from "./components/MultiSelect/useMultiSelectState";
```